deny.toml

```diff
@@ -76,6 +76,8 @@ skip-tree = [
     { name = "pprof" },
     # aws-lc-rs uses a slightly outdated version of bindgen
     { name = "bindgen", version = "0.69.5" },
+    # socket v0.6 is still propagating through the ecosystem
+    { name = "socket2", version = "0.5" },
```

Comment:
pointing this out :point_up_2: tokio 1.47 depends upon socket2@v0.6, so we must note that duplicate socket2@v0.5 dependencies should be ignored by `cargo deny` while this propagates throughout the rest of our dependency tree.